scale/storage/brokers/s3_broker.py

```diff
@@ -121,6 +121,9 @@ def validate_configuration(self, config):
             except ClientError:
                 warnings.append(ValidationWarning('bucket_access',
                                                   'Unable to access bucket. Check the bucket name and credentials.'))
+            except NoCredentialsError:
```

Comment:
You should be able to combine this in the same line as catching the ClientError.